container/root/goss.alpine.yaml

```diff
@@ -36,8 +36,8 @@ package:
     installed: true
   php7-msgpack:
     installed: true
-  php7-memcached:
-    installed: true
+  # php7-memcached:
```

Comment:
Wouldn't you just remove these lines? Or is that commented out in the idea that soon it will come right back?